seed/migrations/0125_dq_refactor.py

```diff
@@ -0,0 +1,98 @@
+# Generated by Django 2.2.10 on 2020-04-13 23:03
+
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    Rule = apps.get_model('seed', 'Rule')
+    # data_type not null:
+    #     data_type is 1:
+    #         text_match null: not_null -> not_null;
+    #                         !not_null -> required: required;
+    #                                   -> !required -> '';
+    #         text_match not null: include -> not_null: (+)not_null;
+    #                                      -> !not_null -> required: (+)required;
+    #     data_type is not 1 (0,2,3,4,5):
+    #         min and max null: not_null -> not_null;
+    #                         : !not_null -> required: required;
+    #                                     -> !required: '';
+    #         min or max not null: range -> not_null: (+)not_null;
+    #                                    -> !not_null -> (+)required: required;
+    # data_type null:
+    #     min and max null -> text_match null: not_null -> not_null;
+    #                                         !not_null -> required: required;
+    #                                                   -> !required: '';
+    #                      -> text_match not null: include -> not_null: (+)not_null;
+    #                                                      -> !not_null -> required: (+)required;
+    #     min or max not null: range -> not_null: (+)not_null;
+    #                                -> !not_null -> required: (+)required;
+
+    def create_condition(rule, condition):
```

Comment:
I agree with you here thanks @adrian-lara !

Reply:
@adrian-lara Here's some before/after cross-check on one of "District of Columbia Department of Energy and Environment":
<img width="1401" alt="Screen Shot 2020-05-05 at 9 44 45 PM" src="https://user-images.githubusercontent.com/49968203/81136981-59fe9680-8f1a-11ea-8290-d1bbebdf4057.png">

<img width="1845" alt="Screen Shot 2020-05-05 at 9 45 13 PM" src="https://user-images.githubusercontent.com/49968203/81136989-5e2ab400-8f1a-11ea-896e-fc156858d195.png">

Get by everything field of a rule EXCPET variables such as severity level and label to avoid duplicates. Create (and update) by not_null/required/min/max/text status.

Reply:
Here are snapshots of before and after migration:
<img width="1845" alt="Screen Shot 2020-05-05 at 9 45 13 PM" src="https://user-images.githubusercontent.com/49968203/81194220-a0371280-8f79-11ea-8bd3-6214137be830.png">
<img width="1401" alt="Screen Shot 2020-05-05 at 9 44 45 PM" src="https://user-images.githubusercontent.com/49968203/81194236-a4fbc680-8f79-11ea-8dba-5667f24fb458.png">